homeassistant/components/watson_tts/tts.py

```diff
@@ -191,7 +191,7 @@ def supported_options(self):
     def get_tts_audio(self, message, language=None, options=None):
         """Request TTS file from Watson TTS."""
         response = self.service.synthesize(
-            message, accept=self.output_format, voice=self.default_voice
+            text=message, accept=self.output_format, voice=self.default_voice
```

Comment:
Looks like https://github.com/home-assistant/core/pull/54916 has the same fix. Unless the library bump is needed for the fix we prefer to separate fixes from other types of changes into different PRs.

Reply:
Sorry. I missed @rutkai fix. Should I close my PR and we'll proceed with this one?

Reply:
It's best to keep the fix separate from the library bump unless the fix requires the library bump. So I suggest we remove the fix here after confirming the above.

Reply:
It's fine to merge your change first, then merging mine would bump the version only - or I'm fine closing my PR. No strings attached ;)

Reply:
I've merged the other PR. Please rebase here on latest dev branch to nly include the library version bump.